common/rtc_manager_rviz_plugin/src/rtc_manager_panel.cpp

```diff
@@ -171,6 +136,20 @@ RTCManagerPanel::RTCManagerPanel(QWidget * parent) : rviz_common::Panel(parent)
   }
   v_layout->addWidget(auto_mode_table_);
 
+  // execution
+  auto * rtc_exe_layout = new QHBoxLayout;
+  {
+    exec_button_ptr_ = new QPushButton("exe mode");
```

Comment:
I think it is better not to abbreviate.
How about following suggestion?

```suggestion
    exec_button_ptr_ = new QPushButton("Execute");
```